src/main/java/org/springframework/integration/kafka/listener/LoggingErrorHandler.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.listener;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.springframework.integration.kafka.core.KafkaMessage;
+
+/**
+ * @author Marius Bogoevici
+ */
+public class LoggingErrorHandler implements ErrorHandler {
+
+	private static final Log LOGGER = LogFactory.getLog(LoggingErrorHandler.class);
+
+	@Override
+	public void handle(Exception thrownException, KafkaMessage message) {
+		LOGGER.error(thrownException);
```

Comment:
No reason in particular. We can do that, too. This was more like a default placeholder implementation.